src/query/interpreter.cpp

```diff
@@ -3068,6 +3067,8 @@ PreparedQuery PrepareSystemInfoQuery(ParsedQuery parsed_query, bool in_explicit_
             {TypedValue("average_degree"), TypedValue(info.average_degree)},
             {TypedValue("memory_usage"), TypedValue(static_cast<int64_t>(info.memory_usage))},
             {TypedValue("disk_usage"), TypedValue(static_cast<int64_t>(info.disk_usage))},
+            // {TypedValue("jemalloc_memory_allocated"),
+            //  TypedValue(utils::GetReadableSize(static_cast<double>(utils::total_memory_tracker.Amount())))},
```

Comment:
Will move this to other PR or maybe leave it here, as it seems useful for end user, what are opinions on this one?